poppy/utils.py

```diff
@@ -683,6 +683,46 @@ def measure_ee(HDUlist_or_filename=None, ext=0, center=None, binsize=None):
     return EE_fn
 
 
+def measure_radius_at_ee(HDUlist_or_filename=None, ext=0, center=None, binsize=None):
+    """ measure encircled energy vs radius and return as an interpolator
+    Returns a function object which when called returns the radius for a given Encircled Energy. This is the
+    inverse function of measure_ee
+
+    Parameters
+    ----------
+    HDUlist_or_filename : string
+        Either a fits.HDUList object or a filename of a FITS file on disk
+    ext : int
+        Extension in that FITS file
+    center : tuple of floats
+        Coordinates (x,y) of PSF center. Default is image center.
+    binsize:
+        size of step for profile. Default is pixel size.
+
+    Returns
+    --------
+    radius: function
+        A function which will return the radius of a desired encircled energy.
+
+    Examples
+    --------
+    EE = measure_radius_at_ee("someimage.fits")
+    print "The EE is 50% at {} arcsec".format(EE(0.5))
+    """
+
+    rr, radialprofile2, EE = webbpsf.radial_profile(HDUlist_or_filename, ext, EE=True, center=center, binsize=binsize)
```

Comment:
Hi @shanosborne, I think on this line you don't need or want to have the `webbpsf.` before the function call, because it's calling a function in this same file `utils.py`.   The unit test is not passing on Travis (see for instance: https://travis-ci.org/mperrin/poppy/jobs/365727304) and it looks like this is why.